### ISSUE TITLE
[BUG] Bootloop after Update 1.10.0

### ISSUE THREAD

**Issue by nillebor** [paperless-ngx/paperless-ngx#2051]
### Description

Paperless-ngx bootloop after update (Watchtower) to 1.10.0.

Bug or new feature?

### Steps to reproduce

Install update
restart paperless

### Webserver logs

```bash
2022-11-28T05:59:42.160441101Z,stdout,LINE 1: CREATE TABLE "django_celery_results_taskresult" ("id" intege...
2022-11-28T05:59:42.160233216Z,stdout,django.db.utils.ProgrammingError: permission denied for schema public
2022-11-28T05:59:42.159981113Z,stdout,    return self.cursor.execute(sql)
2022-11-28T05:59:42.159567917Z,stdout,"  File \"/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py\", line 87, in _execute
2022-11-28T05:59:42.144178156Z,stdout,psycopg2.errors.InsufficientPrivilege: permission denied for schema public
2022-11-28T05:59:42.143923204Z,stdout,    return self.cursor.execute(sql)
2022-11-28T05:59:42.143124850Z,stdout,"  File \"/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py\", line 87, in _execute
```

### Paperless-ngx version

1.10.0

### Host OS

Synology

### Installation method

Docker - official image

**Comment by shamoon** [comment]
The error would imply that the db user youre using doesnt have access to the db e.g. https://duckduckgo.com/?q=psycopg2.errors.InsufficientPrivilege%3A+permission+denied

**Comment by nillebor** [comment]
the error occurred after the update and nothing was changed in the database after setting up paperless-ngx. also other databases run with the same settings

**Comment by shamoon** [comment]
Im not sure what that proves, but I hear you. Did you try any of the suggestions from that search, i.e. granting privileges to the db user?

**Comment by shamoon** [comment]
Did you try any of the suggestions from that search, i.e. granting privileges to the db user?

Do you get the same error if you manually `python3 manage.py migrate` inside the container? And are you using `USERMAP_UID` ?

**Comment by nillebor** [comment]
I could now solve the problem.

I gave the super user rights on the database and restarted postgresql. After that paperless worked. after that i removed the super-rights again and paperless still works.

I think it was due to the restart of postgresql. Until the update paperless and postgresql worked without problems. also restarting the server was without problems.

is it not possible that the update bends the postgresql or connects uncleanly?
